src/dotnet/APIView/APIViewWeb/Models/ReviewModel.cs

```diff
@@ -41,5 +42,39 @@ public bool RunAnalysis
         }
 
         public bool IsClosed { get; set; }
+
+        public HashSet<string> Subscribers { get; set; } = new HashSet<string>();
```

Comment:
I wonder if it should be a dictionary from github login to email so someone can unsubscribe in case email changes. I know it's a long shot so you decide.

Reply:
I guess they would need to know that they can unsubscribe and resubscribe and that we will fix the email for them. If they change emails, it would just show up as unsubscribed and they can subscribe again and we will send to both.

Reply:
Actually, I guess we could just update the email address automatically when they log in if it is different. Seems like it would be a pretty transient and unlikely issue if it ever happened though.